addons/html_editor/static/tests/editor/unbreakable/delete.test.js

```diff
@@ -468,72 +467,66 @@ describe("forward", () => {
-                // // With a div before the list:
-                // await testEditor({
-                //     contentBefore:
-                //         '<div>abc</div><ol><li>def</li><li><div><div>[]ghi</div></div></li><li>jkl</li></ol>',
-                //     stepFunction: deleteBackward,
-                //     contentAfter:
-                //         '<div>abc</div><ol><li>def</li><li><div><div>[]ghi</div></div></li><li>jkl</li></ol>',
-                // });
-            }
-        );
+        test("should not outdent while nested within a list item if the list is unbreakable", async () => {
+            // Only one LI.
+            await testEditor({
+                contentBefore: '<p>abc</p><ol t="1"><li>[]def</li></ol>',
+                stepFunction: deleteBackward,
+                contentAfter: '<p>abc</p><ol t="1"><li>[]def</li></ol>',
+            });
+            // First LI.
+            // await testEditor({
+            //     contentBefore:
+            //         '<ol><li><div><div>[]abc</div></div></li><li>def</li></ol>',
+            //     stepFunction: deleteBackward,
+            //     contentAfter:
+            //         '<ol><li><div><div>[]abc</div></div></li><li>def</li></ol>',
+            // });
+            // // In the middle.
+            // await testEditor({
+            //     contentBefore:
+            //         '<ol><li><div>abc</div></li><li><div><div>[]def</div></div></li><li>ghi</li></ol>',
+            //     stepFunction: deleteBackward,
+            //     contentAfter:
+            //         '<ol><li><div>abc</div></li><li><div><div>[]def</div></div></li><li>ghi</li></ol>',
+            // });
+            // // Last LI.
+            // await testEditor({
+            //     contentBefore:
+            //         '<ol><li>abc</li><li><div><div>[]def</div></div></li></ol>',
+            //     stepFunction: deleteBackward,
+            //     contentAfter:
+            //         '<ol><li>abc</li><li><div><div>[]def</div></div></li></ol>',
+            // });
+            // // With a div before the list:
+            // await testEditor({
+            //     contentBefore:
+            //         '<div>abc</div><ol><li>def</li><li><div><div>[]ghi</div></div></li><li>jkl</li></ol>',
+            //     stepFunction: deleteBackward,
+            //     contentAfter:
+            //         '<div>abc</div><ol><li>def</li><li><div><div>[]ghi</div></div></li><li>jkl</li></ol>',
+            // });
+        });
     });
     describe("selection not collapsed", () => {
+        // @phoenix @todo: this spec is weird. I would argue in favor of
+        // <p class="oe_unbreakable">a[]</p><ol><li>ef</li><li>ghi</li></ol>
         test.todo("shoud not merge list item in the previous unbreakable sibling (1)", async () => {
             await testEditor({
                 contentBefore: unformat(`
-                        <p>a[bc</p>
+                        <p class="oe_unbreakable">a[bc</p>
```

Comment:
Why we don t need the class before ?

Reply:
The original test had the class. When Nicolas reorganized the tests, for some reason the class disappeared. See https://github.com/odoo-dev/odoo/commit/cbfa43786ec8669ef93e98e4e2178e43beb09833#r140126370

Reply:
(without the class, the tests does not match its description...)

Reply:
ok